source/geometries/NextFlexFieldCage.cc

```diff
@@ -330,11 +330,11 @@ void NextFlexFieldCage::DefineMaterials()
 
   // Fiber core material
   if (fiber_mat_name_ == "EJ280") {
-    fiber_mat_ = materials::EJ280();
+    fiber_mat_ = materials::PVT();
     fiber_mat_->SetMaterialPropertiesTable(opticalprops::EJ280());
   }
   else if (fiber_mat_name_ == "EJ286") {
-    fiber_mat_ = materials::EJ280();   // Same base material than EJ280
+    fiber_mat_ = materials::PVT();   // Same base material than EJ280
```

Comment:
By looking at the code PVT is not exactly the same as the EJ280 material you are deleting. Can you explain why?

Reply:
In a previous comment, @jmalbos said that EJ280 is the same as PVT, except for the doping. However, this is not what our code says (the composition in terms of H and C is different). If the code is wrong, please, do eliminate one of the two materials.